lib/api/apiUtils/coldStorage.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Code based on Yutaka Oishi (Fujifilm) contributions
+ * Date: 11 Sep 2020
+ */
+
+/**
+ * Get response header "x-amz-restore"
+ * Be called by objectHead.js
+ * @param {object} objMD - object's metadata
+ * @returns {string} x-amz-restore
+ */
+function getAmzRestoreResHeader(objMD) {
+    let value;
+    if (objMD.archive &&
+        objMD.archive.restoreRequestedAt &&
+        !objMD.archive.restoreCompletedAt) {
+        // Avoid race condition by relying on the `archive` MD of the object
+        // and return the right header after a RESTORE request.
+        // eslint-disable-next-line
+        value = `ongoing-request="true"`;
+    } else if (objMD['x-amz-restore']) {
+        const utcDateTime = new Date(objMD['x-amz-restore']['expiry-date']).toUTCString();
```

Comment:
Move this inside the next if

Reply:
Done: https://github.com/scality/cloudserver/pull/4504/commits/a30ae19648cf97efb17ed42ef27c84bbf91b3fff#diff-5c1a41d999742cdc5b48a81be655c072599261c1755387ef030e91f2f54105b6R23